xml/System.Reflection.Metadata.Ecma335/MetadataBuilder.xml

```diff
@@ -1342,14 +1514,14 @@
         <Parameter Name="methodList" Type="System.Reflection.Metadata.MethodDefinitionHandle" Index="5" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="attributes">To be added.</param>
-        <param name="namespace">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="baseType">To be added.</param>
-        <param name="fieldList">To be added.</param>
-        <param name="methodList">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="attributes">The type attributes.</param>
+        <param name="namespace">The type namespace.</param>
+        <param name="name">The type name.</param>
+        <param name="baseType">The base type entity handle, which can be one of the following: <see cref="T:System.Reflection.Metadata.TypeDefinitionHandle" />, <see cref="T:System.Reflection.Metadata.TypeReferenceHandle" />, <see cref="T:System.Reflection.Metadata.TypeSpecificationHandle" />, or <see langword="null" />.</param>
+        <param name="fieldList">If the type declares fields, set this to the handle of the first one. Otherwise, set this to the handle of the first field declared by the next type definition. If no type defines any fields in the module, <see cref="M:System.Reflection.Metadata.Ecma335.MetadataTokens.FieldDefinitionHandle(System.Int32)" />(1).</param>
+        <param name="methodList">If the type declares methods, the handle of the first one. Otherwise, the handle of the first method declared by the next type definition. If no type defines any methods in the module, <see cref="M:System.Reflection.Metadata.Ecma335.MetadataTokens.MethodDefinitionHandle(System.Int32)" />(1).</param>
```

Comment:
```suggestion
        <param name="methodList">If the type declares methods, the handle of the first one. Otherwise, the handle of the first method declared by the next type definition. If no type defines any methods in the module, <see langword="System.Reflection.Metadata.Ecma335.MetadataTokens.MethodDefinitionHandle(1)" />.</param>
```